python_modules/dagster/dagster/core/test_utils.py

```diff
@@ -438,6 +439,7 @@ def in_process_test_workspace(instance, recon_repo):
 @contextmanager
 def create_test_daemon_workspace():
     """Creates a DynamicWorkspace suitable for passing into a DagsterDaemon loop when running tests."""
+    configure_loggers()
```

Comment:
i know its just for tests but this is kind of a surprising place to see this called as a side effect